src/electrumx/cli/electrumx_compact_history.py

```
@@ -0,0 +1,83 @@
+
+    # When completed also update the UTXO flush count
+    db.set_flush_count(history.flush_count)
+
+
+def main():
+    logging.basicConfig(level=logging.INFO)
+    logging.info('Starting history compaction...')
+    loop = asyncio.get_event_loop()
+    try:
+        loop.run_until_complete(compact_history())
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)